### ISSUE TITLE
Pages missing <title> value

### ISSUE THREAD

**Issue by SWBennett06** [JavaScriptMN/javascriptmn.github.io#18]
There appears to be a bug in the implementation of the [react-helmet](https://github.com/nfl/react-helmet), the package being used to update the document's <title> tag. While the tag exists, it has an empty value. This is problematic for: 

- [accessibility](https://dequeuniversity.com/rules/axe/2.2/document-title?application=lighthouse)
- SEO
- general user experience (as the tab title is simply "javascriptmn.com" no matter the page navigated to)

I'd look into it myself, but the version of react-helmet in place is mildly outdated and the applicable documentation is likely buried in an old commit of the plugin's readme.

**Comment by skylineproject** [comment]
Thanks for opening this, Sean, this issue has been bugging me for some time now. :) 

I'm going to tag this to be completed with the effort to move the site to Gatsby 2.x if that's cool -- I'll try and parcel out some smaller tasks that can make tackling some of these murky dependency updates more manageable. Thanks again!

**Comment by BrianMitchL** [comment]
This was implemented in #31!
